iati/tests/fixtures/versions.py

```diff
@@ -255,6 +262,56 @@ def std_ver_all_uninst_typeerr(request):
     return request.param
 
 
+@pytest.fixture(params=MINOR_MIXED_VER_INVALID + MAJOR_INVALID + iati.tests.utilities.generate_test_types(['int'], True))
+def std_ver_all_uninst_mixederr(request):
+    """Return a value that does not represent a version number."""
+    return request.param
+
+
+@pytest.fixture(params=[
+    ver.iativer_str for ver in iati.version.STANDARD_VERSIONS_MINOR
+] + [
+    ver.semver_str for ver in iati.version.STANDARD_VERSIONS_MINOR
+] + iati.version.STANDARD_VERSIONS_MINOR + [
```

Comment:
This sort of ugly list expansion definitely needs rectifying through use of a function that does it nicely!